.github/workflows/ci.yml

```diff
@@ -22,12 +23,12 @@ jobs:
       - name: Install pnpm
         uses: pnpm/action-setup@v2
         with:
-          version: 8
+          version: 9.12.3
 
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 22.11.0
```

Comment:
Can we just write 22? 
What difference will it make?

Reply:
Yes